runtime/port/unix/j9sysinfo.c

```diff
@@ -1668,6 +1668,16 @@ j9sysinfo_get_cache_info(struct J9PortLibrary *portLibrary, const J9CacheInfoQue
 			omrcpu_get_cache_line_size(&result);
 #endif
 		}
+#elif defined(LINUX) && defined(J9AARCH64)
+	if (query->cmd == J9PORT_CACHEINFO_QUERY_LINESIZE
+	    && query->cacheType == J9PORT_CACHEINFO_DCACHE
+	    && query->level == 1) {
```

Comment:
Formatting nitpick - in vm code we tend to bracket conditionals so no one has to try to figure out order of operations in their heads.
```suggestion
	if ((query->cmd == J9PORT_CACHEINFO_QUERY_LINESIZE)
	&& (query->cacheType == J9PORT_CACHEINFO_DCACHE)
	&& (query->level == 1)
	) {
```

Reply:
Updated the code as suggested.